### PR TITLE
Add PyCon India 2015

### PR DESCRIPTION
**Pull request by MridulS**

### CODE REVIEW

File: pycon-india-2015/videos/discussion-on-building-and-running-communities-pycon-india-2015.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "",
+    "recorded": "",
+    "speakers": [
+        ""
```
Comment (logston) [inline pycon-india-2015/videos/discussion-on-building-and-running-communities-pycon-india-2015.json]: Speakers needed

File: pycon-india-2015/videos/education-education-education-keynote-by-nicholas-htollervey-pycon-india-2015.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "",
+    "recorded": "",
+    "speakers": [
+        ""
```
Comment (logston) [inline pycon-india-2015/videos/education-education-education-keynote-by-nicholas-htollervey-pycon-india-2015.json]: Speaker needed

File: pycon-india-2015/videos/introduction-and-keynote-ajith-kumar-pycon-india-2015.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "",
+    "recorded": "",
+    "speakers": [
+        ""
```
Comment (logston) [inline pycon-india-2015/videos/introduction-and-keynote-ajith-kumar-pycon-india-2015.json]: Speaker needed

File: pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "",
+    "recorded": "",
+    "speakers": [
+        ""
```
Comment (logston) [inline pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json]: Speaker needed

File: pycon-india-2015/videos/lightning-talk-django-deployment-with-git-push-pycon-india-2015.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "Speaker: Prityush",
+    "recorded": "",
+    "speakers": [
+        "Prityus"
```
Comment (logston) [inline pycon-india-2015/videos/lightning-talk-django-deployment-with-git-push-pycon-india-2015.json]: Various speakers

File: pycon-india-2015/videos/lightning-talk-indian-men-are-not-still-rapist-pycon-india-2015.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "",
+    "recorded": "",
+    "speakers": [
+        ""
```
Comment (logston) [inline pycon-india-2015/videos/lightning-talk-indian-men-are-not-still-rapist-pycon-india-2015.json]: Speakers

File: pycon-india-2015/videos/lightning-talk-klonio-pycon-india-2015.json [hunk 7]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "Speaker: Augustus \n\nUrl: https://klonio.com/",
+    "recorded": "",
+    "speakers": [
+        "Augustus "
```
Comment (logston) [inline pycon-india-2015/videos/lightning-talk-klonio-pycon-india-2015.json]: Speakers

File: pycon-india-2015/videos/lightning-talk-pydelhi-dgplug-pycon-india-2015.json [hunk 8]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "Speakers: Anurat, Farhan\n\n- http://pydelhi.org\n- http://dgplug.org/",
+    "recorded": "",
+    "speakers": [
+        " Anurat, Farhan"
```
Comment (logston) [inline pycon-india-2015/videos/lightning-talk-pydelhi-dgplug-pycon-india-2015.json]: Clean up

File: pycon-india-2015/videos/lightning-talk-python-requests-pycon-india-2015.json [hunk 9]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "",
+    "recorded": "",
+    "speakers": [
+        ""
```
Comment (logston) [inline pycon-india-2015/videos/lightning-talk-python-requests-pycon-india-2015.json]: Clean up

File: pycon-india-2015/videos/pretty-printing-in-python-pycon-india-2015.json [hunk 10]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "Speaker:  Shakthi Kannan\n\nThis talk shares the experiments and experiences of using Pronterface and Skeinforge with the Prusia i3 3D printer. Printrun is a free and open source software written in Python for facilitating 3D printing, and is released under GPLv3 license. Skeinforge consists of a set of Python scripts for generating G-Code from 3D models. The Prusia i3 is a RepRap 3D printer design that is also released under the GPLv3 license. The Arduino ATmega2560 controller is used with the printer for configuration, control, and directly interfaces with a computer via USB.\n\nDetails: https://in.pycon.org/cfp/pycon-india-2015/proposals/pretty-printing-in-python/",
+    "recorded": "",
+    "speakers": [
+        " Shakthi Kannan"
```
Comment (logston) [inline pycon-india-2015/videos/pretty-printing-in-python-pycon-india-2015.json]: Space

File: pycon-india-2015/videos/simple-hacks-to-make-your-django-website-faster-pycon-india-2015.json [hunk 11]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "Speaker: Jitendra Agrawal \n\nYou have built an awesome Django website, it's being loved by your early users and you are starting to get some serious traffic. Are you investing more and more time handling alerts, autoscaling and you are starting to get worried about hosting costs? Django framework and Python community has built some great solutions for you.\n\nThis talk will introduce various Django caching features (they are easy to use), talk about AJAX, edge side caching (ESI) and discuss when you should offload some of the tasks to an async process queue so that your users continue to experience a super-fast website.\n\nDetails: https://in.pycon.org/cfp/pycon-india-2015/proposals/simple-hacks-to-make-your-django-website-faster/",
+    "recorded": "",
+    "speakers": [
+        "Jitendra Agrawal "
```
Comment (logston) [inline pycon-india-2015/videos/simple-hacks-to-make-your-django-website-faster-pycon-india-2015.json]: Space

File: pycon-india-2015/videos/sponsored-build-python-web-apps-in-a-breeze-with-visual-studio-pycon-india-2015.json [hunk 12]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "Details: https://in.pycon.org/cfp/pycon-india-2015/proposals/build-python-web-apps-in-a-breeze-with-visual-studio/",
+    "recorded": "",
+    "speakers": [
+        "https://in.pycon.org/cfp/pycon-india-2015/proposals/build-python-web-apps-in-a-breeze-with-visual-studio"
```
Comment (logston) [inline pycon-india-2015/videos/sponsored-build-python-web-apps-in-a-breeze-with-visual-studio-pycon-india-2015.json]: Not a speaker

File: pycon-india-2015/videos/sponsored-contributors-colleagues-clients-customers-open-source-pycon-india-2015.json [hunk 13]
```diff
@@ -0,0 +1,15 @@
+{
+    "description": "Details: https://in.pycon.org/cfp/schedule_item/48/",
+    "recorded": "",
+    "speakers": [
+        "https://in.pycon.org/cfp/schedule_item/48"
```
Comment (logston) [inline pycon-india-2015/videos/sponsored-contributors-colleagues-clients-customers-open-source-pycon-india-2015.json]: Not a speaker

File: pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json (hunk 4 above)
Comment (MridulS) [inline pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json]: name not available on website or in the video.

File: pycon-india-2015/videos/lightning-talk-python-requests-pycon-india-2015.json (hunk 9 above)
Comment (MridulS) [inline pycon-india-2015/videos/lightning-talk-python-requests-pycon-india-2015.json]: Speaker name note available on the website or video

File: pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json (hunk 4 above)
Comment (redapple) [inline pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json]: What I did in other PRs was to use `"Various speakers"`

Comment (redapple) [inline pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json]: At https://youtu.be/dEA-0o7kBzY?t=1m51s , I can see `Kunal Aggarwal`. Other talks by "Kunal" seem to be by the same speaker.

**Comment by redapple** [comment]
@MridulS , could you add a default `"recorded": "2015-10-03"`? The conference took place on October 3rd and 4th 2015 it seems.

File: pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json (hunk 4 above)
Comment (MridulS) [inline pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json]: @redapple Thanks for that :)

File: pycon-india-2015/videos/lightning-talk-python-requests-pycon-india-2015.json (hunk 9 above)
Comment (redapple) [inline pycon-india-2015/videos/lightning-talk-python-requests-pycon-india-2015.json]: I believe it's "Devyani Kota" (@devyanikota)

**Pull request merged**